account/src/test/java/org/killbill/billing/account/api/user/TestDefaultAccountUserApi.java

```diff
@@ -75,20 +77,23 @@ public void testSearch() throws Exception {
         Assert.assertEquals(search1.getMaxNbRecords(), (Long) 2L);
         Assert.assertEquals(search1.getTotalNbRecords(), (Long) 2L);
         Assert.assertEquals(Iterables.toUnmodifiableList(search1).size(), 2);
+        closePagination(search1);
```

Comment:
Unless I'm missing something, I'm pretty sure this isn't needed: `Iterables.toUnmodifiableList` should have consumed the whole stream, so the connection should be automatically closed at this point already (unless there is a bug).